.github/workflows/update-docs.yaml

```diff
@@ -6,7 +6,6 @@ on:
     - main
     - docs
     - 4.*
-    - feature/spring-cloud-azure-passwordless-connection
```

Comment:
Does it mean we can only keep the branch itself for each branch?
For example: In `main` branch, `on.push.branches = main`.